tutorials/joint_1d_3d_analysis.ipynb

```diff
@@ -0,0 +1,1314 @@
+       "                var nbb_cells = Jupyter.notebook.get_cells();\n",
+       "                for (var i = 0; i < nbb_cells.length; ++i) {\n",
+       "                    if (nbb_cells[i].input_prompt_number == nbb_cell_id) {\n",
+       "                        nbb_cells[i].set_text(nbb_formatted_code);\n",
+       "                        break;\n",
+       "                    }\n",
+       "                }\n",
+       "            }, 500);\n",
+       "            "
+      ],
+      "text/plain": [
+       "<IPython.core.display.Javascript object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#%% IEM\n",
+    "iem_fermi = Map.read(\"$GAMMAPY_DATA/catalogs/fermi/gll_iem_v06.fits.gz\")\n",
+    "# Unit is not stored in the file, set it manually\n",
+    "iem_fermi.unit = \"cm-2 s-1 MeV-1 sr-1\"\n",
+    "\n",
+    "# Interpolate the diffuse emission model onto the counts geometry\n",
+    "# The resolution of `diffuse_galactic_fermi` is low: bin size = 0.5 deg\n",
+    "# We use ``interp=3`` which means cubic spline interpolation\n",
+    "data = iem_fermi.interp_by_coord(\n",
+    "    {\"skycoord\": coord.skycoord, \"energy\": coord[\"energy\"]}, interp=3\n",
+    ")\n",
+    "iem_fermi_roi = WcsNDMap(exposure.geom, data, unit=iem_fermi.unit)\n",
+    "model_iem = SkyDiffuseCube(iem_fermi_roi, name=\"iem_v06\")\n",
+    "eval_iem = MapEvaluator(model=model_iem, exposure=exposure, psf=psf_kernel, edisp=edisp)\n",
+    "bkg_iem = eval_iem.compute_npred()\n",
+    "\n",
+    "#%% ISO\n",
+    "filename = \"$GAMMAPY_DATA/fermi_3fhl/iso_P8R2_SOURCE_V6_v06.txt\"\n",
+    "model_iso = create_fermi_isotropic_diffuse_model(\n",
+    "    filename=filename, interp_kwargs={\"fill_value\": None}\n",
+    ")\n",
+    "eval_iso = MapEvaluator(model=model_iso, exposure=exposure, edisp=edisp)\n",
+    "bkg_iso = eval_iso.compute_npred()\n",
+    "\n",
+    "# merge iem and iso  backgrounds for simplicity\n",
+    "background_total = bkg_iso + bkg_iem\n",
+    "background_model = BackgroundModel(background_total)\n",
+    "\n",
+    "background_total.sum_over_axes().smooth(2).plot(\n",
+    "    stretch=\"log\", vmax=50, cmap=\"nipy_spectral\"\n",
+    ");"
+
```

Comment:
I would explain here that we need to create a dataset object by adding a model to the flux points. This might not be obvious to a user.